waddrmgr/db.go

```diff
@@ -432,6 +506,74 @@ func serializeBIP0044AccountRow(encryptedPubKey,
 	return rawData
 }
 
+// fetchAllAccounts loads information about all accounts from the database.
+// The returned value is a slice of account numbers which can be used to load
+// the respective account rows.
+func fetchAllAccounts(tx walletdb.Tx) ([]uint32, error) {
+	accounts := []uint32{}
```

Comment:
I'm now using a 'meta' bucket to store metadata like no of accounts, no of addrs etc and using it to create sized slices.

Moved `bucket := ...` to the first line.